services/openvpn/service/manager.go

```diff
@@ -29,36 +29,42 @@ import (
 	"github.com/mysteriumnetwork/node/identity"
 	"github.com/mysteriumnetwork/node/nat"
 	dto_discovery "github.com/mysteriumnetwork/node/service_discovery/dto"
+	service_openvpn "github.com/mysteriumnetwork/node/services/openvpn"
```

Comment:
lets keep it the same as above 'openvpn_service'

Reply:
Fixed